shell/platform/android/io/flutter/app/FlutterActivity.java

```diff
@@ -151,4 +151,10 @@ public boolean loadIntent(Intent intent) {
     public FlutterView getFlutterView() {
       return flutterView;
     }
+
+    @Override
+    public void onTrimMemory(int level) {
+        if (level == TRIM_MEMORY_RUNNING_LOW)
```

Comment:
Can you add a comment here saying why you selected `TRIM_MEMORY_RUNNING_LOW`? I thought you were going to pick `TRIM_MEMORY_COMPLETE` but realized later that the framework side would never get a callback for this because the application was backgrounded.